src/components/tables/CollectionsTable.vue

```diff
@@ -151,8 +159,8 @@ export default {
 </script>
 
 <style>
-.collection-selection-button {
-  margin-left: 0.75rem;
+.collection-selector-td {
+  padding-right: 1.1em !important;
```

Comment:
!important is a codesmell, please try to solve the problem without for maintainability